content/applications/inventory_and_mrp/quality/quality_management/quality_control_points.rst

```diff
@@ -1,89 +1,123 @@
+
+Begin filling out the new |QCP| by entering a unique :guilabel:`Title` that will make the |QCP|
+easily identifiable.
+
+In the :guilabel:`Products` field, select one or more products that the |QCP| should apply to. If
+the |QCP| should apply to an entire product category, select it in the :guilabel:`Product
+Categories` field.
+
+In the :guilabel:`Operations` field, select the operation(s) that will trigger the |QCP|. For
+example, selecting the *Manufacturing* operation will cause a quality check to be created for new
+|MOs|.
+
+.. note::
+   When creating a new |QCP|, at least one operation must be listed in the :guilabel:`Operations`
+   field. However, the :guilabel:`Products` and :guilabel:`Product Categories` fields can be left
+   blank. If they are, the |QCP| will generate quality checks for every instance of the specified
+   operation(s).
+
+If the *Manufacturing* operation is selected in the :guilabel:`Operations` field, a new field
+appears below it, titled :guilabel:`Work Order Operation`. Select a specific work order to generate
+quality checks for that operation, rather than the manufacturing operation in general.
+
+For example, a |QCP| could be configured to create quality checks for the *Assembly* work order of
+the *Coffee Table* product. When a new |MO| is confirmed for a *Coffee Table*, the |QCP| creates a
+quality check for the *Assembly* operation specifically.
+
+The :guilabel:`Control Per` field is set to one of three options that determine *when* a new quality
+check is created:
+
+- :guilabel:`Operation`: one check is requested for the specified operation as a whole.
+- :guilabel:`Product`: one check is requested for each *unique* product included in the specified
+  operation. For example, a delivery operation for one table and four chairs would generate two
+  checks, since two *unique* products are included in the operation.
+- :guilabel:`Quantity`: a check is requested for a certain percentage of the items within the
+  specified operation. This percentage is set by enabling the :guilabel:`Partial Transfer Test`
+  checkbox, and then entering a numerical value in the :guilabel:`Percentage` field that appears
+  below. If the checkbox is not enabled, one quality check is created for the full quantity.
+
+The :guilabel:`Control Frequency` field is set to one of three options that determine *how often* a
+new quality check is created:
+
+- :guilabel:`All`: a quality check is requested every time the conditions of the |QCP| are met.
+- :guilabel:`Randomly`: a quality check is randomly requested for a certain percentage of
+  operations, which can be specified in the :guilabel:`Every #% of Transfers` field that appears
+  below.
+- :guilabel:`Periodically`: a quality check is requested once every set period of time, which can be
+  specified by entering a numerical value and choosing either :guilabel:`Days`, :guilabel:`Weeks`, or
+  :guilabel:`Months` in the two fields that appear below.
```

Comment:
Reformatted since the line went over 100 char (:
```suggestion
- :guilabel:`Periodically`: a quality check is requested once every set period of time, which can be
  specified by entering a numerical value and choosing either :guilabel:`Days`, :guilabel:`Weeks`,
  or :guilabel:`Months` in the two fields that appear below.
```